src/textual/document/_syntax_aware_document.py

```diff
@@ -82,14 +82,16 @@ def query_syntax_tree(
         Returns:
             A tuple containing the nodes and text captured by the query.
         """
-        captures_kwargs = {}
-        if start_point is not None:
-            captures_kwargs["start_point"] = start_point
-        if end_point is not None:
-            captures_kwargs["end_point"] = end_point
-
-        captures = query.captures(self._syntax_tree.root_node, **captures_kwargs)
-        return captures
+
```

Comment:
This change is more than typing. Can you explain why this change was necessary?

Reply:
This is needed for the new interface in tree-sitter, it's a breaking change in v24